PhysicsTools/NanoAOD/python/jetsAK4_CHS_cff.py

```diff
@@ -169,6 +169,10 @@
     jetTable.variables.puId, doc="Pileup ID flags with 106X (2017) training"
 )
 
+run3_common.toModify(
```

Comment:
same comment about importing `run3_common`

Reply:
Ok, has been updated to solve the issue with the era modifiers that are already imported.